src/Revise.jl

```diff
@@ -319,7 +319,11 @@ function parse_source!(md::ModDict, file::AbstractString, mod::Module, path)
         warn("omitting ", file, " from revision tracking")
         return false
     end
-    parse_source!(md, readstring(file), Symbol(file), 1, mod, path)
+    if VERSION >= v"0.7.0-DEV"
```

Comment:
Sorry, my bad—the Compat PR that includes `read(x, String)` hasn't been merged yet.